### PR TITLE
Change element title to header

### PR DESCRIPTION
**Pull request by CamAnNguyen**
- Change Sample, Wellplate, Screen title to header
- Add pointer cursor to ElemenetAnalysisLabel and ElementCollectionLabel
- Keep ElementLabel in SampleDetail, move label to header in Wellplate and Screen

### CODE REVIEW

File: app/assets/javascripts/components/ElementAnalysesLabels.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ export default class ElementAnalysesLabels extends React.Component {
     return (
       labels.length > 0 ?
         <OverlayTrigger trigger="click" rootClose placement="left" overlay={label_popover}>
-          <span className="collection-label" key={element.id}>
+          <span className="collection-label" key={element.id} style={{cursor: 'pointer'}}>
```
Comment (Sergeykot) [inline app/assets/javascripts/components/ElementAnalysesLabels.js]: I would use 'collection-label' selector in CSS

File: app/assets/javascripts/components/ElementCollectionLabels.js [hunk 2]
```diff
@@ -76,7 +76,7 @@ export default class ElementCollectionLabels extends React.Component {
       labels.length > 0 ?
         <OverlayTrigger trigger="click" rootClose placement="left"
                         overlay={label_popover}>
-          <span className="collection-label" key={element.id}>
+          <span className="collection-label" key={element.id} style={{cursor: 'pointer'}}>
```
Comment (Sergeykot) [inline app/assets/javascripts/components/ElementCollectionLabels.js]: I would use 'collection-label' selector in CSS
 Add a line note

**Comment by Sergeykot** [comment]
@CamAnNguyen

**Comment by CamAnNguyen** [comment]
Thank you for this idea. Updated

**Comment by Sergeykot** [comment]
:+1:

**Comment by PiTrem** [comment]
only partially resolve #492

**Pull request merged**